staging-service-shared/src/main/java/com/github/cafdataprocessing/services/staging/dao/filesystem/BatchNameProvider.java

```diff
@@ -17,13 +17,18 @@
 
 import com.github.cafdataprocessing.services.staging.BatchId;
 import com.github.cafdataprocessing.services.staging.utils.ServiceIdentifier;
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
 import java.time.Instant;
+import java.time.ZoneOffset;
 import java.time.format.DateTimeParseException;
+import java.util.Date;
+import java.util.TimeZone;
```

Comment:
We shouldn't be using any of these legacy date types.  You should be using the modern Date Time API (i.e. in the `java.time.` package.

- https://docs.oracle.com/javase/tutorial/datetime/iso/legacy.html